### ISSUE TITLE
Bug 407: Importing / cloning data from existing to new indexes

### ISSUE THREAD

**Issue by handrews** [GrandComicsDatabase/gcd-django#59]
http://dev.comics.org/bugs/show_bug.cgi?id=407
Reporter: @adia 

When indexing reprint collections, it would be much more efficient to import
existing data from the reprinted issues. We need a way to do this easily.

One way to do this would be to add an export function, so that one can then
import the exported file to the new index.

A better way would be to tie this into the reprint link system. For example,
when adding sequences to a new issue, maybe there should be a way to add
reprinted stories.

An efficient work-flow for adding multiple reprinted stories might go like
this: first, you visit all reprinted issues one by one, and add all the
required sequences to the cache. Then, while editing the new index, there is a
new button labeled "add reprinted stories". Without adding any stories to the
new index, you press it, and you are presented with a list of all cached
stories. You can now tick all you need, submit the list, and copies of the
original sequences appear in the new index.

This would benefit from the implementation of bug #312 (reordering sequences
should be easier).

An option to copy the existing data to empty fields when adding a reprint to an
existing sequence would also be useful.

------- Comment #1 From Jochen G. 2010-05-15 23:08:04 PST [reply] -------
I think this should be independent from reprints, but similar in workflow.

Sometimes you may just want to copy from a similar story. You still should be
able to generate a reprint link at the same time if you want to.

So the button should read something like 'add copy of existing story'. How to
select this story can be quite similar (if not the same) as the procedure to
select the story for matching reprints. I tried to keep the code for that as
modular as possible and need to rework it a bit more to have it flexible in
this regard.

In other words, once reprint editing/matching/selecting is done quite a bit of
the code for copying story data is there.

------- Comment #2 From Jochen G. 2012-09-20 20:54:21 PST [reply] -------
**\* Bug 755 has been marked as a duplicate of this bug. ***

------- Comment #3 From Jochen G. 2013-08-20 04:04:08 PST [reply] -------
**\* Bug 817 has been marked as a duplicate of this bug. ***

------- Comment #4 From Peter Croome 2013-08-20 11:41:04 PST [reply] -------
I'm just adding a few keywords here so that when I decide again that I like
this idea, this bug comes up in my search... since obviously I can't be trusted
to remember :)

duplicate sequence
duplication
copying sequence
